src/main/java/uk/gov/hmcts/probate/controller/DataExtractController.java

```diff
@@ -31,6 +32,7 @@ public class DataExtractController {
     private final ExelaDataExtractService exelaDataExtractService;
     private final SmeeAndFordDataExtractService smeeAndFordDataExtractService;
     private final DataExtractDateValidator dataExtractDateValidator;
+    private final DormantCaseService dormantCaseService;
```

Comment:
Why have we added DormantCaseService here - does not look like its being used?

Reply:
Removed